### PR TITLE
Add another span to illustrate usage

### PR DESCRIPTION
**Pull request by DanielMSchmidt**

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
screen shot? looks good though!

**Comment by DanielMSchmidt** [comment]
bildschirmfoto 2017-11-09 um 09 17 54
bildschirmfoto 2017-11-09 um 09 18 03

File: react-native-example/App.js [hunk 1]
```diff
@@ -1,40 +1,56 @@
...
-      .then(response => response.json())
-      .then(person => this.setState({
-        person: person.name,
-      })
-      ).catch(e => console.error(e));
+    const id = tracer.createRootId();
+
+    tracer.scoped(() => {
+      tracer.setId(id);
+      tracer.recordAnnotation(new Annotation.ServerRecv());
```
Comment (codefromthecrypt) [inline react-native-example/App.js]: ps for "local spans" we tend to use client annotations. You might want to add a TODO or even better implement this! https://github.com/openzipkin/zipkin-js/issues/104

**Comment by DanielMSchmidt** [comment]
@adriancole I see, changed it like you said

**Comment by codefromthecrypt** [comment]
thx!

**Comment by DanielMSchmidt** [comment]
Awesome!

**Pull request merged**